src/main/java/mate/academy/spring/service/impl/RoleServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package mate.academy.spring.service.impl;
+
+import mate.academy.spring.dao.RoleDao;
+import mate.academy.spring.model.Role;
+import mate.academy.spring.service.RoleService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class RoleServiceImpl implements RoleService {
+    private RoleDao roleDao;
+
+    public RoleServiceImpl(RoleDao roleDao) {
+        this.roleDao = roleDao;
+    }
+
+    @Override
+    public Role add(Role role) {
+        return roleDao.add(role);
+    }
+
+    @Override
+    public Role getByName(String roleName) {
+        return roleDao.getByName(roleName);
```

Comment:
```suggestion
        return roleDao.getByName(roleName).orElseThrow(
                () -> new RuntimeException("Can't get role: "
                        + roleName)
        );
```
In Dao we retun Optional, so here we need to return this way